core-processor/src/configs.rs

```diff
@@ -26,7 +26,7 @@ const MAX_WASM_PAGES: u32 = 512;
 const INIT_COST: u64 = 5000;
```

Comment:
Those numbers should be updated and ultimately benchmarked

Now it is too low

Reply:
Append new issue for this: https://github.com/gear-tech/gear/issues/895